nav2_util/include/nav2_util/bt_loop_rate.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2024 Angsa Robotics
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rclcpp/rclcpp.hpp"
+#include "behaviortree_cpp/bt_factory.h"
+#include "behaviortree_cpp/behavior_tree.h"
+
+namespace nav2_util
+{
+
+class LoopRate
```

Comment:
`BTLoopRate` --> move to the `nav2_behavior_tree` package. I don't want the BT.CPP dependencies to stretch too low considering the utils are used by other non-Nav2 packages & we have a nice separation of Nav2 BT utilities in the `nav2_behavior_tree` package already.

The naming to make more clear, but also if you move to the `nav2_behavior_tree::LoopRate` namespace, we can drop the `BT` prefix since the namespace indicates that this is a behavior tree specific object.